configure

```diff
@@ -1,28 +1,32 @@
 #!/bin/sh
 
-# Locate python2 interpreter and re-execute the script.  Note that the
-# mix of single and double quotes is intentional, as is the fact that
-# the ] goes on a new line.
+# Locate an acceptable python interpreter and then re-execute the script.
+# Note that the mix of single and double quotes is intentional,
+# as is the fact that the ] goes on a new line.
 _=[ 'exec' '/bin/sh' '-c' '''
+test ${TRAVIS} && exec python "$0" "$@"  # workaround for pyenv on Travis CI
```

Comment:
Could this comment be more informative? I think it should be
```
# On Travis, pyenv arranges for the requested python version to be available as
# plain "python". In other environments, prefer the version specific executable names.
```
But I might misunderstand its purpose.